docker/docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
 
   nbxplorer:
     restart: unless-stopped
-    image: ghcr.io/elenpay/nbxplorer:v2.3.65-1.0.0
+    image: ghcr.io/elenpay/nbxplorer:elenpay-develop
```

Comment:
why are we doing this?

Reply:
Pointing to a fixed version is wrong, we lost features with this typo.